src/Domain.LinnApps/Stock/StockPool.cs

```diff
@@ -0,0 +1,79 @@
+﻿namespace Linn.Stores2.Domain.LinnApps.Stock
+{
+    using System;
+
+    public class StockPool
+    {
+        public StockPool()
+        {
+        }
+
+        public StockPool(
+            string code,
+            string description,
+            string dateInvalid,
+            AccountingCompany accountingCompany,
+            int? sequence,
+            string stockCategory,
+            int? defaultLocation,
+            StorageLocation storageLocation,
+            int? bridgeId,
+            string availableToMrp)
+        {
+            this.StockPoolCode = code.ToUpper().Trim();
+            this.StockPoolDescription = description;
+            this.DateInvalid = Convert.ToDateTime(dateInvalid);
+            this.AccountingCompany = accountingCompany;
+            this.Sequence = sequence;
+            this.StockCategory = stockCategory;
+            this.DefaultLocation = defaultLocation;
+            this.StorageLocation = storageLocation;
+            this.BridgeId = bridgeId;
+            this.AvailableToMrp = availableToMrp;
+        }
+
+        public string StockPoolCode { get; protected set; }
```

Comment:
i like protected, but i'd either use it for all these setters or not at all, just for consistency